ferrilata-devilline/Properties/launchSettings.json

```diff
@@ -1,29 +1,12 @@
 {
-    "iisSettings": {
-        "windowsAuthentication": false,
-        "anonymousAuthentication": true,
-        "iisExpress": {
-            "applicationUrl": "http://localhost:56788",
-            "sslPort": 44329
-        }
-    },
-    "profiles": {
-        "IIS Express": {
-            "commandName": "IISExpress",
-            "launchBrowser": true,
-            "environmentVariables": {
-                "ASPNETCORE_ENVIRONMENT": "Development",
-                "FDTOKENSECRET": "6c07d2e1-3395-4780-8c4a-900f58202043"
-            }
-        },
-        "ferrilata_devilline": {
-            "commandName": "Project",
-            "launchBrowser": true,
-            "environmentVariables": {
-                "ASPNETCORE_ENVIRONMENT": "Development",
-                "FDTOKENSECRET": "6c07d2e1-3395-4780-8c4a-900f58202043"
-            },
-            "applicationUrl": "https://localhost:5001;http://localhost:5000"
-        }
+  "profiles": {
+    "ferrilata_devilline": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
```

Comment:
Please add  "FDTOKENSECRET": "6c07d2e1-3395-4780-8c4a-900f58202043" to env variables for "ferrilata_devilline" profile